src/Tuura/Concept/FSM/Translation.hs

```diff
@@ -1,3 +1,23 @@
+-----------------------------------------------------------------------------
```

Comment:
Why don't you add proper Haddock headers, like here: 

https://github.com/snowleopard/alga/blob/master/src/Algebra/Graph.hs#L4-L8

List yourself as the maintainer and provide contact details. Perhaps, you could even release it on Hackage, so that it can be used by others as a library and to have rendered docs available online.